build.gradle

```diff
@@ -325,48 +324,63 @@ signing {
  *
  * For releasing to your local maven repo, use gradle install
  */
-uploadArchives {
-    repositories {
-        mavenDeployer {
-            beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-
-            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-                authentication(userName: project.findProperty("sonatypeUsername"), password: project.findProperty("sonatypePassword"))
-            }
-
-            snapshotRepository(url: "https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/") {
-                authentication(userName: System.env.ARTIFACTORY_USERNAME, password: System.env.ARTIFACTORY_PASSWORD)
-            }
-
-            pom.project {
-                name 'Picard'
-                packaging 'jar'
-                description 'A set of command line tools (in Java) for manipulating high-throughput sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF.'
-                url 'http://broadinstitute.github.io/picard/'
-
-                developers {
-                    developer {
-                        id 'picard'
-                        name 'Picard Team'
-                        url 'http://broadinstitute.github.io/picard'
-                    }
-                }
+publishing {
+    publications {
+        picard(MavenPublication) {
+            from components.java
+
+            artifact sourcesJar
```

Comment:
Future note when updating gradle build on its own.
We probably want to consider using the newer

```
java {
   withSourcesJar()
   withJavadocJar()
}
```
That handles attaching them to the publishing configuration automatically.